Sources/MapboxNavigation/CPRouteChoice.swift

```diff
@@ -0,0 +1,34 @@
+import CarPlay
+import MapboxDirections
+
+@available(iOS 12.0, *)
+extension CPRouteChoice {
+    
+    struct RouteResponseUserInfo {
```

Comment:
Do we expect these structs in the user info dictionary to be accessible to application code? If so, we’ll need to apply access control keywords and potentially add an initializer.

Reply:
There is no intention to make them public at this point, so I guess we can omit any modifications to make them more accessible.